src/main/java/com/chartiq/finsemble/example/JavaExample.java

```diff
@@ -378,6 +391,29 @@ private void setFormEnable(boolean enabled) {
         dockingCheckbox.setDisable(!enabled);
     }
 
+    private void handleGetComponentStateCb(JSONObject err, JSONObject res) {
+        if(err!=null){
+            fsbl.getClients().getLogger().error(err.toString());
+        }else{
+            //Set subscribe linker channel
+            JSONArray channelToLink = res.getJSONArray("Finsemble_Linker");
+            for(int i=0;i<channelToLink.length();i++){
+                Button tempBtn = (Button) window.getScene().lookup("#"+channelToLink.getString(i)+"Button");
+                Platform.runLater(() -> tempBtn.fire());
+                fsbl.getClients().getLogger().log(channelToLink.getString(i));
+            }
+
+            //Set symbol value
+            String symbol = res.getString("symbol");
+            if(!symbol.equals("")){
+                Platform.runLater(() -> symbolLabel.setText(symbol));
+            }
```

Comment:
You need to check whether `res` has the property "symbol" before reading it.
```suggestion
            if (res.has("symbol")) {
                final String symbol = res.getString("symbol");
                if(!symbol.equals("")) {
                    Platform.runLater(() -> symbolLabel.setText(symbol));
                }
            }
```